master/internal/experiment.go

```diff
@@ -395,6 +395,34 @@ func (e *experiment) Receive(ctx *actor.Context) error {
 
 		ctx.Log().Info("experiment shut down successfully")
 
+	case *apiv1.PostSearcherOperationsRequest:
+		queue, err := e.searcher.GetCustomSearcherEventQueue()
+		if err != nil {
+			ctx.Log().WithError(err).Errorf("Custom Searcher Event Queue was not retrieved.")
+		}
+		// Ensuring that the lastProcessedEvent is the same as what the
+		// (triggered) client processed and the sent the appropriate opertations.
+		// if queue.GetLastProcessedEventId() == -1 then the
+		// first event initial operations triggered the client.
+		if msg.TriggeredByEvent.Id == queue.GetLastProcessedEventID() ||
```

Comment:
Wouldn't `msg.TriggeredByEvent.Id` rather be the first _unprocessed_ event? I.e. the event immediately _after_ the last processed event?

Reply:
oh yes that's true. Thanks for catching that.